Assets/HoloToolkit/Input/Tests/Scripts/SimpleGridGenerator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class SimpleGridGenerator : MonoBehaviour
```

Comment:
Just a comment to clarify this:
>> If the desire is to keep HoloToolkit simple, I would suggest adding a separate Examples/Test folder under the root HoloToolkit folder instead of keeping Test folders alongside each module.

We started like this but then we had cross folder dependencies which was not fun. This way developers can completely delete the folder feature areas that they don't rely on.